multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -103,21 +103,41 @@ const toOracleOfferId = (idx: number) => `oracle${idx + 1}-accept`;
 
 test.serial('oracles accept', async t => {
   const { oracleWds, retryUntilCondition, vstorageClient, wallets } = t.context;
+  const brands = await vstorageClient.queryData('published.agoricNames.brand');
+  const { Invitation } = Object.fromEntries(brands);
 
-  const instances = await vstorageClient.queryData(
-    'published.agoricNames.instance',
-  );
-  const instance = fromEntries(instances)[contractName];
+  const description = 'oracle operator invitation';
+
+  // ensure we have an unused (or used) oracle invitation in each purse
+  let hasAccepted = false;
+  for (const name of keys(oracleMnemonics)) {
+    const { offerToUsedInvitation, purses } = await vstorageClient.queryData(
+      `published.wallet.${wallets[name]}.current`,
+    );
+    const { value: invitations } = balancesFromPurses(purses)[Invitation];
```

Comment:
Does that work?

`balancesFromPurses` returns a map, right? and `Invitation` is a `Brand`